test/WatchTestCases.test.js

```diff
@@ -76,9 +77,7 @@ describe("WatchTestCases", () => {
 						"js",
 						"watch-src",
 						category.name,
-						`${testName}-${Math.random()
-							.toPrecision(21)
-							.slice(2)}`
+						testName
```

Comment:
If I remember right, this was crashing `jest` when there were multiple copies of packages. That's why I added the extra random suffix.